lib/features/pokemon_details/pokemon_details_page.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:pokedex_async_redux/api/model/model.dart';
+import 'package:pokedex_async_redux/features/pokemon_details/widgets/widgets.dart';
+import 'package:pokedex_async_redux/utils/utils.dart';
+import 'package:flutter/material.dart';
+
+class PokemonDetailsPage extends StatelessWidget {
+  const PokemonDetailsPage({
+    required this.pokemonDetails,
+    required this.pokemonName,
+    required this.pokemonImage,
+    super.key,
+  });
+
+  final Async<PokemonDetails> pokemonDetails;
+  final String pokemonName;
```

Comment:
use the whole pokemon model instead, then place it above `pokemonDetails` property 🙂